### ISSUE TITLE
Windows 64bit binaries

### ISSUE THREAD

**Issue by Alex-PLACET** [libexpat/libexpat#139]
Currently you only provide Windows 32 binaries on [SourceForge](https://sourceforge.net/projects/expat/files/expat_win32/). I see that the 64bit build is already ready in the Appveyor job. Do you plan to release it ?

**Comment by hartwork** [comment]
You're the first to ask. Is compiling from sources an option or is lack of 64bit binaries a showstopper to you?

**Comment by Alex-PLACET** [comment]
Compiling from sources is a option and it works, but I'm comfortable with the idea to use and distribute binaries from an official build :)

**Comment by pepone** [comment]
We [ZeroC](https://github.com/zeroc-ice) build expat libraries for Win32 and x64 platforms and are available as NuGet packages

[expat.v141](https://www.nuget.org/packages/expat.v141/2.2.4)
[expat.v140](https://www.nuget.org/packages/expat.v140/2.2.4)
[expat.v120](https://www.nuget.org/packages/expat.v120/2.2.4)

**Comment by hartwork** [comment]
For the record, there's a button/link `Manual download` up there to download a `.nupkg` file that's actually a ZIP including this content (subset):
```
/build                  
  /native               
    /bin                
      /Win32            
        /Debug          
          /libexpatd.dll
          /libexpatd.pdb
        /Release        
          /libexpat.dll 
          /libexpat.pdb 
      /x64              
        /Debug          
          /libexpatd.dll
          /libexpatd.pdb
        /Release        
          /libexpat.dll 
          /libexpat.pdb 
    /expat.v141.targets 
    /include            
      /expat.h          
      /expat_external.h 
    /lib                
      /Win32            
        /Debug          
          /libexpatd.lib
        /Release        
          /libexpat.lib 
      /x64              
        /Debug          
          /libexpatd.lib
        /Release        
          /libexpat.lib 
```

**Comment by hartwork** [comment]
I have marked this ticket as a _question_ now and am going to close it as _answered_.

**Comment by hartwork** [comment]
A quick update for anyone interested in 64bit Expat Windows binaries:

[Expat 2.6.4](https://github.com/libexpat/libexpat/releases/tag/R_2_6_4) is the first release to come with 64bit Windows binaries. These binaries were built by GitHub Actions at https://github.com/libexpat/libexpat/actions/runs/12878310769 based on the work from pull request #946. If you find any issues with these new 64bit binaries specifically, please [report that as a new issue](https://github.com/libexpat/libexpat/issues) and we'll see about a fix for Expat >=2.6.5, together. Thank you!
